### ISSUE TITLE
Irarutu is not SHWNG

### ISSUE THREAD

**Issue by king-ben** [lexibank/abvdoceanic#49]
https://youtu.be/33B3FXQmvok?t=336

**Comment by maryewal** [comment]
As far as I can tell they haven't published this yet, but it is convincing. 
Since SHWNG is just our out-group here, and we aren't focused on membership in it, I would say just remove it and avoid entering the discussion. @RustyGray and @SimonGreenhill, thoughts?

**Comment by king-ben** [comment]
I agree, especially because its deletion can most likely be justified based on existing literature which says how uncertain it is.
